Editor/Helper/ButtonMethodScriptableObjectEditor.cs

```diff
@@ -0,0 +1,56 @@
+/// <summary>
+/// ButtonMethodAttribute,
+/// modified from https://github.com/Deadcows/MyBox/blob/master/Attributes/ButtonMethodAttribute.cs
+/// </summary>
+using UnityEngine;
+
+namespace UnityAddressableImporter.Helper.Internal
+{
+	using System;
+	using System.Collections.Generic;
+	using System.Reflection;
+	using Editor.Helper;
+	using UnityEditor;
+
+
+	[CustomEditor(typeof(AddressableImportSettings), true), CanEditMultipleObjects]
+	public class ButtonMethodScriptableObjectEditor : Editor
```

Comment:
The class and this filename shall be renamed to `AddressableImportSettingsEditor`.